tests/include/detray/test/utils/simulation/event_generator/random_track_generator_config.hpp

```diff
@@ -52,7 +52,10 @@ struct random_track_generator_config {
 
     /// Track origin
     darray<scalar_t, 3> m_origin{0.f, 0.f, 0.f};
-    darray<scalar_t, 3> m_origin_stddev{0.f, 0.f, 0.f};
+    darray<scalar_t, 3> m_origin_stddev{
```

Comment:
BTW, what is an advantage of setting default values to `epsilon`? clients will need to change this to (0,0,0) manually when they want to fix the vertex.

Reply:
My compiler throws an assertion in release mode when 0 is passed as stddev. So I cannot run any test/tool that involves the random track generation

Reply:
The Gaussian distribution is not directly defined for stddev = 0 mathematically, so I guess that the implementation in the standard library calculates the sample value by dividing by the standard deviation somewhere, so that this would result in an FPE? I changed our method now, so that it returns the mean when the standard dev is zero without calling the dist